test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -378,26 +351,21 @@ def test_t5(self):
 
         # Verify that the accounting logs have Resource_List.<resource> but no
         # R records.
-        m = self.server.accounting_match(
+        self.server.accounting_match(
             msg='.*Resource_List.*', id=jid1, regexp=True)
-        self.assertNotEqual(m, None)
-        m = self.server.accounting_match(
-            msg='.*R;' + jid1 + '.*resources_used.*', id=jid1, regexp=True)
-        self.assertIsNone(m)
-        m = self.server.accounting_match(
+        msg = '.*R;' + jid1 + '.*resources_used.*'
+        self.server.accounting_match(msg, id=jid1, regexp=True,
+                                     existence=False)
```

Comment:
@bhroam Thanks for review. There is no difference in call. I just formatted little bit. Also I removed 'msg=' from call as first argument to accounting_match is message and I am still passing that using msg variable.